app/models/user.rb

```diff
@@ -24,6 +24,9 @@ class User < ApplicationRecord
   belongs_to :current_group, :class_name => "MiqGroup"
   has_and_belongs_to_many :miq_groups
   scope      :admin, -> { where(:userid => "admin") }
```

Comment:
This does not feel right, as anyone with `MiqUserRole::SUPER_ADMIN_ROLE_NAME` can be considered admin as well.

Let me see if I can get rid of this.